install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -538,15 +538,22 @@ data:
   enable-auto-protect-node-port-range: {{ .Values.nodePort.autoProtectPortRange | quote }}
 {{- end }}
 {{- if hasKey .Values "loadBalancer" }}
+{{- if .Values.loadBalancer.standalone }}
+  datapath-mode: lb-only
+{{- end }}
 {{- if hasKey .Values.loadBalancer "mode" }}
-  node-port-mode: {{ .Values.loadBalancer.mode | quote }}
```

Comment:
Are we deprecating this option? if yes it needs to be added in the upgrade notes

Reply:
Same as ^^.